### PR TITLE
Refactor hero, area weserve, bio section to new layout

### PR DESCRIPTION
**Pull request by NurkaAmre**
## Context

[link to Jira card](https://ywgc.atlassian.net/jira/software/projects/YWGC/boards/1?selectedIssue=YWGC-44)

## Work done

- [ ] Hero section photo now takes up half of the page instead of a third
- [ ] Add a new photo to a bio section
- [ ] Add any new icons in the new design to these sections
- [ ] Update the background color of sections to the new design
- [ ] Update section header horizontal rule to new design

## Manual testing instructions

- [ ] Mobile/Desktop responsive
- [ ] Accessibility
- [ ] Vercel

## Gotchas/What I learned
Hide and seek

### CODE REVIEW

**Review by ajkarczewski — APPROVED** [review]
Nice job! Looks great so far

**Comment by alliequintano** [comment]
I noticed the image is looking low-resolution and @ajkarczewski will update figma with higher quality files. Not-blocking for this PR, we can open a new PR with the updated images once we have them.

File: src/components/ServiceArea.jsx [hunk 1]
```diff
@@ -2,14 +2,22 @@ import React from 'react'
...
-    <div className='bg-gray-100'>
-      <div className='px-6 pt-10'>
-        <h2 className='text-center mb-3 text-5xl md:text-6xl font-header'>Areas We Serve</h2>
-        <hr className='w-[180px] md:w-[720px] h-px mx-auto mt-2 mb-10 md:mb-2 bg-gray-500 border-0 rounded dark:bg-gray-700' />
-        <p className='text-xl my-10 text-left'>We are proudly located in Detroit, Michigan. We service the <span className='font-bold'>Detroit, Highland Park, Hamtramck,</span> and <span className='font-bold'>metro Detroit </span>area. Unsure if we serve your location? Give us a call!
-        </p>
+    <div className='flex flex-col md:flex-row'>
+      <div className='pt-10 md:flex-2 md:flex-shrink-1 md:flex-grow-0 md:w-1/2 justify-center items-center md:px-24 md:pt-16 md:pb-32'>
+        <h2 className='text-center mb-3 text-5xl md:text-6xl font-header'>Where We Serve</h2>
+        <img src='../images/dog.png' className='w-[100%] h-[200px] object-contain pt-16 mb-5 md:pt-10' />
```
Comment (novellac) [inline src/components/ServiceArea.jsx:8]: Issue: Missing alt text.

File: src/components/ServiceArea.jsx [hunk 2]
```diff
@@ -2,14 +2,22 @@ import React from 'react'
...
+        <div className='text-sm md:mt-10 flex flex-col md:justify-between'>
+          <p className='mb-10 md:my-10 px-6 md:px-0'>We are proudly located in Detroit, Michigan. We service the Detroit, Highland Park, Hamtramck, and metro Detroit area.
+          </p>
+          <iframe src="https://www.google.com/maps/d/embed?mid=1DG3IBrt1W1CSnau3KX33eHbSTV1uHtU&ehbc=2E312F" className='md:hidden px-0 h-[263px]' aria-label='Service Area Map'></iframe>
```
Comment (novellac) [inline src/components/ServiceArea.jsx:12]: Question: Should an `iframe` use `aria-label` or `title` (or something else)? Let's make sure we're using the right attributes for the element.

File: src/components/ServiceArea.jsx [hunk 3]
```diff
@@ -2,14 +2,22 @@ import React from 'react'
...
+          <p className='pt-10 md:pt-0 text-center md:text-start'>Unsure if we service your area?</p>
+        </div>
+        <div className='flex md:gap-2 flex-row pb-16 md:text-start text-xl md:2xl font-semibold pl-14 md:pl-0 md:mt-2'>
+          <h2>Call us today!</h2>
+          <p className='text-green-800'>123.456.7890</p>
+        </div>
       </div>
-      <iframe src="https://www.google.com/maps/d/embed?mid=1DG3IBrt1W1CSnau3KX33eHbSTV1uHtU&ehbc=2E312F" className='w-full h-96' aria-label='Service Area Map'></iframe>
+      <iframe src="https://www.google.com/maps/d/embed?mid=1DG3IBrt1W1CSnau3KX33eHbSTV1uHtU&ehbc=2E312F" className='md:block hidden md:flex-2 md:flex-shrink-1 md:flex-grow-0 md:w-1/2' aria-label='Service Area Map'></iframe>
```
Comment (novellac) [inline src/components/ServiceArea.jsx:20]: Same question as above.

File: src/pages/Home.jsx [hunk 4]
```diff
@@ -7,86 +7,84 @@ function Home () {
...
-              <div className='w-[358px] h-[80px] px-8'>
-                <h2 className='font-normal italic text-center text-3xl text-zinc-500'>
-                  Detroit&apos;s Finest Lawn Care
-                </h2>
+              <div className='px-16 gap-2.5 pt-10'>
+                <BaseHeading level={1} text='Yorkshire Woods Ground Crew' />
+              </div>
+              <div>
+                <div className='px-8'>
+                  <h2 className='font-normal italic text-center text-3xl text-zinc-500'>
```
Comment (novellac) [inline src/pages/Home.jsx:26]: Question (nb, can refactor later): Why use `BaseHeading` in some places and heading tags in others?

File: src/pages/Home.jsx [hunk 5]
```diff
@@ -7,86 +7,84 @@ function Home () {
...
+                  </h2>
+                </div>
               </div>
             </div>
-            <div className='gap-16'>
-              <div className='flex flex-col font-normal text-xl text-center mb-14 gap-2'>
+            <div className='flex flex-col font-normal text-xl text-center mb-14 gap-2 py-10.5'>
                 <h3 className=''>Personalized Service</h3>
                 <h3 className=''>Jobs Of All Sizes</h3>
                 <h3 className=''>Quality Guaranteed</h3>
```
Comment (novellac) [inline src/pages/Home.jsx:35]: Question: Why are these heading tags? What content are they headings for?

File: src/pages/Home.jsx [hunk 6]
```diff
@@ -7,86 +7,84 @@ function Home () {
...
               <h3 className='px-2'>Jobs Of All Sizes</h3>
               <h3 className='px-2'>Quality Guaranteed</h3>
             </div>
               <Link
                 to='#services-section'
-                className='text-2xl lg:text-3xl text-white font-bold'
+              className='text-2xl lg:text-3xl text-white font-bold'
               >
-            <div className='flex items-center text-center justify-center bg-green-700 p-2.5 gap-2.5 rounded-lg w-3/5 lg:w-[463px] lg:h-[85px]'>
+              <div className='flex h-16 md:px-20 items-center text-center justify-center bg-green-700 self-stretch rounded-full transition-all hover:text-green-700 hover:bg-transparent hover:border-solid hover:border-4 hover:border-green-700'>
```
Comment (novellac) [inline src/pages/Home.jsx:81]: Issue: A `div` has no semantic meaning, but the text inside looks meaningful. What would be a better tag for this? My guess is a heading tag of some level.

**Review by novellac — CHANGES_REQUESTED** [review]
Please add specific manual testing instructions.

File: src/pages/Home.jsx (hunk 5 above)
Comment (NurkaAmre) [reply src/pages/Home.jsx:35]: @novellac Maybe that question for a designer?

**Review by ajkarczewski — APPROVED** [review]

**Pull request merged**